### PR TITLE
[WIP] temporal-pca-flip

### PR DESCRIPTION
**Pull request by SherazKhan**
Just need to write test!

### CODE REVIEW

**Comment by SherazKhan** [comment]
Added test.

File: examples/inverse/plot_label_source_activations.py [hunk 1]
```diff
@@ -48,6 +48,7 @@
 mean = stc.extract_label_time_course(label, src, mode='mean')
 mean_flip = stc.extract_label_time_course(label, src, mode='mean_flip')
 pca = stc.extract_label_time_course(label, src, mode='pca_flip')
+pca_mean = stc.extract_label_time_course(label, src, mode='pca_flip_mean')
```
Comment (Eric89GXL) [inline examples/inverse/plot_label_source_activations.py]: you should add it to the plots below

File: mne/source_estimate.py [hunk 2]
```diff
@@ -2769,6 +2769,9 @@ def _gen_extract_label_time_course(stcs, labels, src, mode='mean',
     elif mode == 'pca_flip':
         # get the sign-flip vector for every label
         label_flip = _get_label_flip(labels, label_vertidx, src[:2])
+    elif mode == 'pca_flip_mean':
+        # get the sign-flip vector for every label
+        label_flip = _get_label_flip(labels, label_vertidx, src[:2])
```
Comment (Eric89GXL) [inline mne/source_estimate.py]: new `mode` needs to be added to public-facing docstring in the appropriate places, you can `git grep pca_flip` to figure out where

Comment (SherazKhan) [inline mne/source_estimate.py]: Perfect will do it this week.

**Comment by dengemann** [comment]
What we need here is:

- [ ] a better name
- [ ] some conceptual documentation
- [ ] examples / tutorials / comparisons
- [ ] tests

**Comment by agramfort** [comment]
@SherazKhan any hope to complete this?

**Comment by dengemann** [comment]
Alex I think we're trying to understand more systematically what this
temporal pca flip actually does. Don't expect this to be closed too soon.
We want to merge things we understand and recommend.

File: mne/source_estimate.py [hunk 3]
```diff
@@ -2830,13 +2830,9 @@ def _gen_extract_label_time_course(stcs, labels, src, mode='mean',
...
                     # determine sign-flip
-                    sign = np.sign(np.dot(U[:, 0], flip))
-
-                    # use average power in label for scaling
-                    scale = linalg.norm(s) / np.sqrt(len(vertidx))
-                    flip_refrence = sign * scale * V[0]
+                    flip_refrence = V[0]
                     flip = np.sign(np.corrcoef(flip_refrence, stc.data[vertidx, :]))[1:,0]
-                    label_tc[i] = np.mean(flip[:,np.newaxis] * stc.data[vertidx, :], axis=0)
+                    label_tc[i] = np.median(flip[:,np.newaxis] * stc.data[vertidx, :], axis=0)
```
Comment (dengemann) [inline mne/source_estimate.py]: That was the difference?

**Comment by larsoner** [comment]
@SherazKhan do you think we still need this? Are you using it in production code / publications, or have you found the existing (or other) methods to work fine?

**Comment by larsoner** [comment]
Closing for now then, feel free to reopen if it's still desired

**Pull request closed without merging**